### DISCUSSION TITLE
short URL shown in the address bar not pressing short URL redirect me to the long ones

### DISCUSSION THREAD

**Discussion by sameh-yasuo** [YOURLS/YOURLS#3352] [Plugins]
I used curl for that but gives me only html without CSS its for security and original URL masking

  $ch = curl_init($location);
        curl_setopt($ch, CURLOPT_REFERER, $location);

        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
        curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, FALSE);
        $content = curl_exec($ch);
        curl_close($ch);
        echo ($content);
        return 1;

**Comment by ozh** [comment]
Not a single sentence here makes sense.

**Reply by sameh-yasuo** [reply]
It's not an issue..
Just wanted from the short url to stay  in the browser just like you said in your description the issue #363

**Reply by ozh** [reply]
It may be not an issue, it still needs to make sense if you want people to read, understand and help.

If I understand correctly : look at the bundled plugin "YOURLS Toolbar"

**Reply by sameh-yasuo** [reply]
am sorry for not being clear still a student but am asking for the toolbar plugin when I activated it there is an error connecting to the original URL  maybe because am using yourls version in localhost?

**Reply by dgw** [reply]
Localhost has a lot of restrictions on it these days. Even if you use a browser with more lenient rules for local content, the long URL's server can send a header denying permission to display its content in an iframe.

The true cause of your particular error should be evident from the console (usually shown by pressing F12)

**Reply by ozh** [reply]
What does "connecting to the URL" mean ? You don't "connect" to an URL.

"There is an error" : what error ? When doing what exactly ?

Vague information will get you nowhere.

**Reply by sameh-yasuo** [reply]
@dgw  maybe that's true I need a domain okay thank you :)

![errroooor](https://user-images.githubusercontent.com/57291152/170871057-bd26e69f-4176-4c2e-8ab3-053986f7a10c.png)

**Reply by ozh** [reply]
The toolbar plugin works fine on localhost. It's just that most sites don't want to be included in an iframe.

Bottom line : toolbars suck.

![image](https://user-images.githubusercontent.com/223647/170871794-b5670140-b4b1-4345-bb14-27c8dbbc4aa3.png)

**Reply by sameh-yasuo** [reply]
yes  there is some sites works fine like your site ozh.org but I want to configure a solution to that, so I change the url-redirect function without the  toolbar plugin it gives me html only for all sites with a short URL in the browse
![codeeeeee](https://user-images.githubusercontent.com/57291152/170872244-dde0ca6a-98bb-40c0-8c8f-21f3c6341250.png)
it  

this is what I
![websiteTest](https://user-images.githubusercontent.com/57291152/170872423-0c38b142-82d7-4e54-b0b9-b5370365bee3.png)
 get
